src/frontend/komponenter/Fagsak/Behandlingsresultat/Behandlingsresultat.tsx

```diff
@@ -45,8 +45,8 @@ const EndretUtbetalingAndel = styled.div`
     margin-bottom: 1rem;
 `;
 
-const StyledEditIkon = styled(Edit)`
-    margin-right: 0.5rem;
+const StyledEditIkon = styled(PencilIcon)`
+    font-size: 1.5rem;
```

Comment:
Tok vek margin-right fordi det ga unaturlig mye spacing. Lot ikonet være litt mindre enn det forrige fordi jeg syns det forrige var litt for stort og dette er "default" størrelse
Før
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/b912b0e3-48f7-4b48-9471-ab55fb4553cc)

Etter
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/485fc5bc-51aa-4ec8-a3df-e4fe5a31efa4)